contrib/copy-tex/copy-tex.js

```diff
@@ -0,0 +1,71 @@
+// Set these to how you want inline and display math to be delimited.
+const copyDelimiters = {
+    inline: ['$', '$'],    // alternative: ['\(', '\)']
+    display: ['$$', '$$'], // alternative: ['\[', '\]']
+};
+
+// Replace .katex elements with their TeX source (<annotation> element).
+// Modifies fragment in-place.  Useful for writing your own 'copy' handler.
+window.katexReplaceWithTex = function(fragment) {
```

Comment:
`katexReplaceWithTex` doesn't need to go on `window`.  In fact this whole file could be wrapped in an IIFE to avoid polluting the global namespace.

Reply:
The whole thing is already wrapped in an IIFE because it gets babelified into `build/contrib/copy-tex*.js`.  I was polluting the namespace in case someone wanted to build their own copy handler using this function.  In hindsight, that's not a very nice way to do it.

Instead I've split the function into its own module, `katex2tex.js`, which is the re-usable part.  (This is similar to how `auto-render` does it.)  Babelify merges these two files into one.  No more namespace pollution.

You still can directly change the delimiters only by manually editing the build.  (This is necessary if we don't want to expose any pollution.)  But now you could copy/paste the rather short `copy-tex.js` into your own build environment (and `require('katex/contrib/copy-tex/katex2tex.js')`).

Reply:
right... I forgot we started babelifying the contrib folder.